docs/api-testing/use-cases/fact.md

```diff
@@ -0,0 +1,298 @@
+---
+id: fact
+title: Using Fact for Handling Email Notifications
+sidebar_label: Improving Email Notifications using Fact
+description: Using Fact for Handling Email Notifications
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The **Fact** component is used to control the behaviour of Email notifications, which (if enabled) alert you to test failures. It enables you to add information about the nature of the test execution.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+- Your email address already saved in the [Notifications](/api-testing/project-access/#email-notifications) section.
+- Familiarity with the [API Testing Composer](/api-testing/composer/).
+
+## Setting Alert Environments
+
+Testing activity is tracked using test ID number. This may not work if you're testing in multiple environments, as an incident could be environment-specific. When a Fact component is added to a test, it will inform our system which environment the execution relates to so that the incident signature will carry the environment as well.
```

Comment:
```suggestion
Testing activity is tracked using a test ID number. This may not work if you're testing in multiple environments, as an incident could be environment-specific. When a Fact component is added to a test, it will inform our system which environment the execution relates to so that the incident signature will carry the environment as well.
```